chroma-manager/tests/framework/services/cluster_setup

```diff
@@ -22,13 +22,13 @@ yum install -y git python-virtualenv python-setuptools python-devel gcc make gra
 
 # Create a user so we can run chroma as non-root
 if ! id chromatest; then
-    useradd chromatest
+    useradd -b /localhome chromatest
```

Comment:
This seems to be rather environment specific.  How is this going to affect people trying to run this test outside of the environment this change is being made for?